src/graphql/account.rs

```diff
@@ -1121,7 +1136,8 @@ mod tests {
                         password: "Ahh9booH",
                         role: "SECURITY_ADMINISTRATOR",
                         name: "John Doe",
-                        department: "Security"
+                        department: "Security",
+                        language: "English"
```

Comment:
Why don't we change this value to what GraphQL clients will actually use?

Reply:
Since the UI uses the combination of ISO 639-1 and ISO 3166-1 codes such as "en-US", the value Graphql clients use will be the string like "en-US". I have just replace `"English"` and `"Korean"` with `"en-US"` and `"ko-KR"`. Did I understand your points right?

Reply:
Looks awesome.